spark-bigquery-dsv1/spark-bigquery-with-dependencies-parent/pom.xml

```diff
@@ -50,6 +50,9 @@
             <relocation>
               <pattern>com.github</pattern>
               <shadedPattern>com.google.cloud.spark.bigquery.repackaged.com.github</shadedPattern>
+              <excludes>
```

Comment:
is it possible to check if we can just remove this pattern altogether?  If not please add a comment on why we are doing the exclusion.

Reply:
As of now we can remove the pattern altogether. My only concern with that is if start using another library with the `com.github` pattern in the future and accidentally include it here, the ZSTD implementation will break again. Excluding it specifically is much safer in that case. 

Let me know if that makes sense. Otherwise, happy to remove the pattern completely as well.

Reply:
That is fair, I think the probability of this is very low.  We could ensure things don't break by having appropriate integration tests.